src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
Hmm... for Python we're 0.15.x already. I don't remember where this is in the code, but @geigerj knows.

My main concern about the version number is that it does not convey that this is a beta quality code generator (in terms of what it produces). My subjective reaction is that it tells me that we just started working on this project a couple months ago.

My bigger concern is making sure we advance this version number each time we generate a new version of GAPIC. Otherwise the metrics for which versions of GAPIC are being used for each language won't be particularly interesting.